src/tiledb/cloud/geospatial/ingestion.py

```diff
@@ -845,14 +845,13 @@ def ingest_raster_udf(
                                 chunk_window, dst.transform
                             )
 
-                            for b in range(dst.count):
-                                chunk_arr, _ = rasterio.merge.merge(
-                                    input_datasets,
-                                    bounds=chunk_bounds,
-                                    nodata=nodata,
-                                    indexes=[b + 1],
-                                )
-                                dst.write(chunk_arr, window=chunk_window)
+                            chunk_arr, _ = rasterio.merge.merge(
+                                input_datasets,
+                                bounds=chunk_bounds,
+                                nodata=nodata,
+                            )
+                            dst.write(chunk_arr, window=chunk_window)
```

Comment:
`chunk_arr` as output of one method and input of the next helps ensure raster shape fidelity.

Reply:
A reason for iterating over the bands was in the case of ingesting HSI data to reduce memory footprint. However the tile/cube sizes are relatively small, I was over thinking it as this is a generic ingestor.

Reply:
@normanb I'll restore that, then. I don't want to cause a regression.